bin/postgres-ha/bootstrap-postgres-ha.sh

```diff
@@ -98,15 +99,15 @@ initialization_monitor() {
             then
                 # Apply enhancement modules
                 echo_info "Applying enahncement modules"
-                for module in /opt/cpm/bin/modules/*.sh
+                for module in "${CRUNCHY_DIR}/bin/modules/*.sh"
```

Comment:
This was globbing before. The `*` needs to be outside any quotes. Something like:

```suggestion
                for module in "${CRUNCHY_DIR}"/bin/modules/*.sh
```